rclcpp_components/include/rclcpp_components/component_manager.hpp

```diff
@@ -23,20 +23,26 @@
 
 #include "class_loader/class_loader.hpp"
 
-#include "rclcpp/executor.hpp"
-#include "rclcpp/node_options.hpp"
-#include "rclcpp/rclcpp.hpp"
-
 #include "composition_interfaces/srv/load_node.hpp"
 #include "composition_interfaces/srv/unload_node.hpp"
 #include "composition_interfaces/srv/list_nodes.hpp"
 
+#include "rclcpp/executor.hpp"
+#include "rclcpp/node_options.hpp"
+#include "rclcpp/rclcpp.hpp"
+
 #include "rclcpp_components/node_factory.hpp"
+#include "rclcpp_components/visibility_control.hpp"
+
+namespace class_loader
```

Comment:
Since `ClassLoader` is already included, is it necessary to forward declare it here?

Reply:
sorry, I've forgotten to commit that change. Wanted to remove the include here.